pennylane/transforms/insert_ops.py

```diff
@@ -221,7 +221,7 @@ def f(w, x, y, z):
     if not isinstance(op_args, Sequence):
         op_args = [op_args]
 
-    num_preps = sum(isinstance(o, STATE_PREP_OPS) for o in circuit.operations)
+    num_preps = sum(o.base_name in state_prep_ops for o in circuit.operations)
```

Comment:
Could one not ask how many queue to `_prep` here?

Reply:
Yes.  Good catch.